layouts/shortcodes/codepen.html

```diff
@@ -0,0 +1,9 @@
+<p data-height="{{ .Get "height" | default "573" }}"
+   data-theme-id="light"
+   data-slug-hash="{{ .Get "slug"}}"
+   data-default-tab="result"
+   data-user="{{ .Get "user" | default "Serabe" }}"
+   data-embed-version="2"
+   data-pen-title="{{ .Get "title" | default "sourced"}}"
+   class="codepen"></p>
+<script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
```

Comment:
**important** 
By the way: `MathJax` script should be moved into [`partial/js.html`](https://github.com/src-d/blog/blob/new-design/themes/hugo-steam-theme/layouts/partials/js.html) instead of being in [`partial/footer.html`](https://github.com/src-d/blog/blob/new-design/themes/hugo-steam-theme/layouts/partials/footer.html#L27), what seems to be wrong